application/aam-backend-service/src/test/resources/database/documents/ReportConfig:2.json

```diff
@@ -2,5 +2,6 @@
   "_id": "ReportConfig:2",
   "title": "Test Report 2",
   "mode": "sql",
-  "aggregationDefinition": "SELECT s.name as name, s.privateSchool as privateSchool FROM School as s"
+  "aggregationDefinition": "SELECT s.name as name, s.privateSchool as privateSchool FROM School as s",
+  "args": []
```

Comment:
can we add a test cases including from + to args here (both for testing and as implicit documentation of how the query looks like).